src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExporter.cs

```diff
@@ -17,75 +17,78 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.IO;
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
-using Dynatrace.OpenTelemetry.Exporter.Metrics;
 using OpenTelemetry.Metrics.Export;
 
-namespace Dynatrace.OpenTelemetry.Exporter
+namespace Dynatrace.OpenTelemetry.Exporter.Metrics
 {
     /// <summary>
     /// https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/metric-ingestion-protocol/
     /// https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/post-ingest-metrics
     /// </summary>
     public class DynatraceMetricsExporter : MetricExporter
     {
-        internal readonly DynatraceExporterOptions Options;
-        private readonly ILogger<DynatraceMetricsExporter> logger;
-        private HttpClient httpClient;
-        private DynatraceMetricSerializer serializer;
+        private readonly DynatraceExporterOptions _options;
+        private readonly ILogger<DynatraceMetricsExporter> _logger;
+        private readonly HttpClient _httpClient;
+        private readonly DynatraceMetricSerializer _serializer;
 
         public DynatraceMetricsExporter(DynatraceExporterOptions options = null, ILogger<DynatraceMetricsExporter> logger = null)
         {
-            this.Options = options ?? new DynatraceExporterOptions();
-            this.logger = logger ?? NullLogger<DynatraceMetricsExporter>.Instance;
+            this._options = options ?? new DynatraceExporterOptions();
+            this._logger = logger ?? NullLogger<DynatraceMetricsExporter>.Instance;
             logger.LogDebug("Dynatrace Metrics Url: {Url}", options.Url);
```

Comment:
```suggestion
            logger.LogDebug("Dynatrace Metrics Url: {Url}", this._options.Url);
```
Could be null otherwise.